src/hooks/tests/use-stop-watch.test.ts

```diff
@@ -0,0 +1,121 @@
+import { act, renderHook } from '@testing-library/react-hooks';
+import { withFakeTimers } from '../../helpers/test';
+import { useStopWatch } from '../use-stop-watch';
+
+describe('useStopWatch', () => {
+  const setup = () => {
+    const { result } = renderHook(() => useStopWatch());
+    return result.current;
+  };
+
+  it('should getElapsedTime after starting', () => {
+    withFakeTimers(async () => {
```

Comment:
you'll need a return here since `withFakeTimers` is asynchronous (implied bc the inner callback is async)
alternatively, you can promote the outer scope to async and await it but the former (just returning) is better and equivalent

you need to do this bc jest awaits each test, and if you're not returning the promise, the test finishes before the promise resolves which leaks into other tests

the same applies to all tests in this file as well as `use-timer.test.ts`, not sure why I didn't catch that oops

Reply:
huh. had no clue, lol. But that makes sense. It should be fixed now.